pkg/backend/httpstate/client/client.go

```diff
@@ -305,6 +305,52 @@ type serviceTokenInfo struct {
 	Team         string `json:"team,omitempty"`
 }
 
+//nolint:gosec // These are token type URNs, not credentials
```

Comment:
Might be worth pointing out these are Uniform Resource Name URI URN's not Unique Resource Name URNs that we see over most of the rest of the codebase.